src/actions.ml

```diff
@@ -1550,6 +1563,77 @@ let coq_bug_minimizer_results_action ~bot_info ~ci ~key ~app_id body =
         Server.respond_string ~status:(`Code 400) ~body:"Bad request" ()
   else Server.respond_string ~status:(`Code 400) ~body:"Bad request" ()
 
+let coq_bug_minimizer_resume_ci_minimization_action ~bot_info ~key ~app_id body
+    =
+  if string_match ~regexp:"\\([^\n]+\\)\n\\([^\r]*\\)" body then
+    let stamp = Str.matched_group 1 body in
+    let message = Str.matched_group 2 body in
+    match Str.split (Str.regexp " ") stamp with
+    | [comment_thread_id; _author; _repo_name; _branch_name; owner; repo] -> (
+        message |> String.split ~on:'\n'
+        |> function
+        | docker_image
+          :: target
+             :: opam_switch
+                :: failing_urls
+                   :: passing_urls :: base :: head :: bug_file_lines ->
+            (let bug_file_contents = String.concat ~sep:"\n" bug_file_lines in
+             fun () ->
+               init_git_bare_repository ~bot_info
+               >>= fun () ->
+               Github_installations.action_as_github_app ~bot_info ~key ~app_id
+                 ~owner ~repo
```

Comment:
I'm a bit surprised to see this here instead of in the call to the action function from `bot.ml`.

Reply:
I was copying https://github.com/coq/bot/blob/e7327b50da0d8aeaa68cd4ebb137cef0224aa233/src/actions.ml#L1523-L1534
Should both of these be moved to bot.ml?

Reply:
Probably yes.

Reply:
Oh, the issue is that we don't have access to owner and repo from `bot.ml`; they are only accessible after we parse the stamp

Reply:
Oh I see.